### PR TITLE
qtemu: init at 2.1

### PR DESCRIPTION
**Pull request by romildo**
###### Motivation for this change

Add [QtEmu](https://gitlab.com/qtemu/gui).

> QtEmu is a graphical user interface for QEMU written in Qt. 
It has the ability to run virtual operating systems on native systems. 
This way you can easily test a new operating system or try a Live CD on your system without any troubles and dangers.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by privateseabass** [comment]
works for me.

File: pkgs/applications/virtualization/qtemu/default.nix [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    qemu
+  ];
+
+  installPhase = ''
+    runHook preInstall
+    install -D -t $out/share/applications qtemu.desktop
+    install -D -t $out/share/pixmaps qtemu.png
+    install -D -t $out/bin qtemu
+    runHook postInstall
+  '';
```
Comment (Ma27) [inline pkgs/applications/virtualization/qtemu/default.nix:30]: Doesn't take the build-system care of the install?

**Review by Ma27 — CHANGES_REQUESTED** [review]
Please wrap the binaries to make sure that the `qemu-*` executables are found.

File: pkgs/applications/virtualization/qtemu/default.nix (hunk 1 above)
Comment (romildo) [reply pkgs/applications/virtualization/qtemu/default.nix:30]: No, the upstream build-system lacks an install method.

**Comment by romildo** [comment]
> Please wrap the binaries to make sure that the `qemu-*` executables are found.

Done.

**Comment by romildo** [comment]
@Ma27 Is it ok now?

**Pull request merged**